prow/github/client.go

```diff
@@ -2588,6 +2588,33 @@ func (c *client) UpdateRepo(owner, name string, repo RepoUpdateRequest) (*FullRe
 		requestBody: &repo,
 		exitCodes:   []int{200},
 	}, &retRepo)
+
+	return &retRepo, err
+}
+
+// UpdateRepoTopics replaces the topics on an existing repository
+// See https://developer.github.com/v3/repos/#edit
+func (c *client) UpdateRepoTopics(owner, name string, repo RepoUpdateRequest) (*FullRepo, error) {
```

Comment:
```suggestion
// See https://developer.github.com/v3/repos/#edit
func (c *client) UpdateRepoTopics(owner, name string, topics RepoTopicsRequest) error {
```
My read of the docs is that the array of topics is returned, not the full repo.

I'm not intimately familiar with the conventions of prow's github client, but looking at some of the other `UpdateFoo` methods you probably just want to return an error if you fail

I'd also recommend adding a new type to types.go, since there's nothing else here that uses an anonymous struct, e.g.
```golang
// RepoTopicsRequest represents topics for a repo
// See also: https://docs.github.com/en/rest/repos/repos#replace-all-repository-topics
type RepoTopicsRequest struct {
	Names []string `json:"names"`
}
```